api/v1/api.md

```diff
@@ -0,0 +1,151 @@
+# dcrtime API Specificiation
+
+V1
+
+This document describes the REST API provided by a `dcrtimed` server.  This API allows users to create upload hashes which are periodically submitted to the decred blockchain and timestamped as well as to confirm such timestamps.
+
```

Comment:
We need to add something like "and validate digests inclusion."  Or something; basically something that describes the merkle root/path validation.  This is a major feature that we want mentioned in the header.